storage/bootstrap/bootstrapper/fs/source.go

```diff
@@ -116,11 +116,7 @@ func (s *fileSystemSource) enqueueReaders(
 	readersCh chan<- shardReaders,
 ) {
 	for shard, tr := range shardsTimeRanges {
-		var files []string
-		fs.ForEachInfoFile(s.filePathPrefix, namespace, shard, s.readerBufferSize, func(fname string, _ []byte) {
-			files = append(files, fname)
-		})
-
+		files := fs.ReadInfoFiles(s.filePathPrefix, namespace, shard, s.readerBufferSize)
```

Comment:
Ah I see, misread it somehow :\ although the new change makes it read the info data even though it doesn't need to, which adds a bit of overhead.